client/api_challenge.go

```diff
@@ -159,10 +193,32 @@ func (c *Client) SubmitChallenge(ctx context.Context, challengerAddress, spOpera
 	return resp.TxResponse, nil
 }
 
-// Attest handles user's request for attesting a challenge.
-// The attestation can include a valid challenge or is only for heartbeat purpose.
-// If the challenge is valid, the related storage provider will be slashed.
-// For heartbeat attestation, the challenge is invalid and the storage provider will not be slashed.
+// AttestChallenge will send the attestation result of a challenge. The attestation can include a valid challenge or is only for heartbeat purpose.
```

Comment:
We should follow this format
// AttestChallenge - Brief Introduction
//
// Detailed Introduction, need to describe only validators should call this API

Reply:
Fixed format. Added more information.